columnflow/production/cms/dy.py

```diff
@@ -0,0 +1,458 @@
+# coding: utf-8
+
+"""
+Column production methods related to Drell-Yan reweighting.
+"""
+
+from __future__ import annotations
+
+import law
+
+from dataclasses import dataclass
+
+from columnflow.production import Producer, producer
+from columnflow.util import maybe_import, InsertableDict
```

Comment:
Just a single comment because I just tested this: the `columnflow.util.InsertableDict` has been removed at some point, apparently
```suggestion
from columnflow.util import InsertableDict
from law.util import InsertableDict
```